essentials.yml

```diff
@@ -1,6 +1,7 @@
 ---
 ath:
-  athRevision: "dad333092159cb368efc2f9869572f0a05d255ac"
+  athRevision: "JENKINS-50790-3"
+  athUrl: "https://github.com/raul-arabaolaza/acceptance-test-harness.git"
```

Comment:
You should be able to just specify the hash of the head of your PR, right? GitHub lets you check out a commit from any fork in the network. Depends on exactly how the `checkout` command is structured: if you

```groovy
git 'https://github.com/jenkinsci/acceptance-test-harness'
sh "git checkout $athRevision"
```

then it will not work, since the cloned refs do not include PRs by default. But if you use the `checkout` step with an appropriately configured reference it should work.

Reply:
```
checkout changelog: true, poll: false, scm: [$class           : 'GitSCM', branches: [[name: athRevision]],
                                                                 userRemoteConfigs: [[url: athUrl]]]
``` 
Was not aware of GitHub able to search also forks for commits, that's great to know, thanks!

Reply:
In any case this PR was never intended to be merged